packages/google_maps_flutter/android/src/main/java/io/flutter/plugins/googlemaps/GoogleMapController.java

```diff
@@ -199,6 +199,13 @@ public void onMethodCall(MethodCall call, MethodChannel.Result result) {
           markersController.changeMarkers((List<Object>) markersToChange);
           Object markerIdsToRemove = call.argument("markerIdsToRemove");
           markersController.removeMarkers((List<Object>) markerIdsToRemove);
+          result.success(null);
+          break;
+        }
+      case "map#stateSnapshot":
```

Comment:
IMO it would be more consistent with other plugins to have a separate async MethodCall name for each property that you want to read off of the googleMap and expose this data as a public API of the plugin. This gives developers a suite of useful API for getting for properties of the map that can also be used in integration testing.